README.md

```diff
@@ -0,0 +1,24 @@
+# Project 6: Spots
+
```

Comment:
* Please, make the modals match the design (`mobile` and `desktop`). The design can be found here https://www.figma.com/design/1qCS9RkiKiVquBhpOJqjZ0/Sprint-5-Project%3A-Spots?node-id=0-1&node-type=canvas&t=Y8QUtuwYfAUZYJn2-0 . Pay attention, that there are 2 designs: `mobile` and `desktop`. They have different font sizes, margins, paddings, close icon positions and so on